<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>639</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog { border-image: url(:/newPrefix/tela-05.png) 0 0 0 0 stretch stretch;}</string>
  </property>
  <widget class="QLabel" name="barraLateral">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 102, 106);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBg">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>561</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisc">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVoltar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoSigla">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : #F0F2F2;
	background : rgba(240, 242, 242, 1);
	border-radius : 6px;
	-moz-border-radius : 6px;
	-webkit-border-radius : 6px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoDescricao">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : #F0F2F2;
	background : rgba(240, 242, 242, 1);
	border-radius : 6px;
	-moz-border-radius : 6px;
	-webkit-border-radius : 6px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(83, 83, 83);</string>
   </property>
   <property name="text">
    <string>Sigla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(83, 83, 83);</string>
   </property>
   <property name="text">
    <string>Descrição</string>
   </property>
  </widget>
  <zorder>labelBg</zorder>
  <zorder>barraLateral</zorder>
  <zorder>label</zorder>
  <zorder>btnHelp</zorder>
  <zorder>btnDisc</zorder>
  <zorder>btnVideo</zorder>
  <zorder>btnHome</zorder>
  <zorder>labelDisc</zorder>
  <zorder>btnVoltar</zorder>
  <zorder>campoSigla</zorder>
  <zorder>campoDescricao</zorder>
  <zorder>btnSalvar</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="tela.qrc"/>
  <include location="texto.qrc"/>
  <include location="tela_disc.qrc"/>
 </resources>
 <connections/>
</ui>
